src/os/bluestore/StupidAllocator.cc

```diff
@@ -139,10 +141,11 @@ int StupidAllocator::allocate(
     }
   }
 
-  // search down (origin)
+  // search down (from origin, and skip searched extents by hint)
```

Comment:
@jjhuo i concur with you. in theory we should return `-ENOSPC` here, before searching in the smaller bins in vain.